APIs/schemas/endpoint.json

```diff
@@ -72,8 +72,8 @@
     },
     "max_bandwidth": {
       "type": "string",
-      "pattern": "^(?:0.|[1-9][0-9]*[.])?[0-9]+[kMG]bit\/s$",
-      "description": "Optional. Maximum bandwidth that the sender is allowed to send. Format as numbers followed by kbit/s, Mbit/s, Gbit/s."
+      "pattern": "^(0|[1-9][0-9]*)(\.[0-9]+)?[kMG]bit\/s$
```

Comment:
```suggestion
      "pattern": "^(0|[1-9][0-9]*)(\.[0-9]+)?[kMG]bit\/s$",
```

Reply:
Thanks. Glad someone is paying attention :-)